src/components/partner/dto/partner-type.enum.ts

```diff
@@ -0,0 +1,11 @@
+import { registerEnumType } from '@nestjs/graphql';
+
+export enum PartnerType {
```

Comment:
This should be merged with `PartnershipType`....probably this one kept

Reply:
So I have to move `PartnershipType` to `partner-type.enum.ts`, right?

Reply:
What I mean is they are the same enum, so we don't need both. I think it makes sense for the type to be here since this is the "source" from the business' perspective. That means we should delete `PartnershipType` and replace usage with `PartnerType`

Reply:
Got it, thanks